doc/manual/manual.tex

```diff
@@ -10818,6 +10818,125 @@ \subsubsection{Yamauchi \& Takei anelastic shear wave velocity-temperature conve
 
 
 
+\subsubsection{Compaction -- differentiation}
+
+
+\textit{This section was contributed by Marine Lasbleis.}
+
+Here we use \aspect{} to mode the compaction of a simple column with a mixture of liquid and solid using two-phase flow equations. 
+It presents a modification of the global melt material model to use different parametrization of the permeability, shear viscosity and bulk viscosity. 
+
+We model the compaction of a mixture of liquid and fluid, without additional melting and freezing.  
+This set-up has been used in the PhD thesis of Ondrej Sramek (\cite{SRPHD07}) as an analog for the differentiation of an initially undifferentiated planet where the solid matrix is the silicates and the liquid the molten iron. 
+We test here using the same set of equations than in \cite{SRB07} (SRB07), but without melting and freezing and in the limit of $\omega=0$
+and the liquid viscosity negligible compared to the solid one. 
+Both phases are considered to be viscously deformed by the compaction, and the liquid is lighter than the solid. 
+
+The plugin and parameter files can be found at \url{benchmarks/differenciation/melt_srb.cc} and \url{benchmarks/differenciation/diff_adim.prm}. 
```

Comment:
need to adjust the directory names here and in the line below

Reply:
done